doc/article/technical_article.tex

```diff
@@ -188,15 +194,20 @@ \subsection{Node.js}
 
 \subsection{Vue.js}
 Vue.js is a progressive framework for building user interfaces.\cite{vuejs}
-Vue.js is used for trckr mainly for its simplicity and the rather shallow
-learning curve it provides to unexperienced developers. The features that Vue.js
-provides allow the creation of data structures that can easily be displayed in
-on a website. This and the ability to easily make calls to the backend make it a
-good fit for trckr.
+Vue.js is used mainly for its simplicity and the rather shallow learning curve
+it provides unexperienced developers. The features that Vue.js provides allow
```

Comment:
Try not to use "prodives" twice in a row
Rather something like: "The features of Vue.js allow..."